enterprise/api_client/enterprise_catalog.py

```diff
@@ -43,22 +46,57 @@ def create_enterprise_catalog(
             'enabled_course_modes': enabled_course_modes,
             'publish_audit_enrollment_urls': json.dumps(publish_audit_enrollment_urls),
         }
-        return endpoint.post(post_data)
+        try:
+            return endpoint.post(post_data)
+        except (SlumberBaseException, ConnectionError, Timeout) as exc:
+            LOGGER.exception(
+                'Failed to create EnterpriseCustomer Catalog [%s] in enterprise-catalog due to: [%s]',
+                catalog_uuid, str(exc)
+            )
+            return False
```

Comment:
i guess my personal preference is to raise here anyway to force callers to deal with error scenarios, but i don't have a strong opinion on this.

Reply:
Yeah I don't love just silently failing and only having the logs, but it also felt weird to me for them to hit save, get an error, and then have their changes actually take effect. Open to others' opinions on this